src/pages/CreateClub/CreateClub.tsx

```diff
@@ -24,9 +25,10 @@ export const CreateClub: React.FC = () => {
   });
   const [isSubmitted, setIsSubmitted] = useState(false);
   const [id, setId] = useState<number | null>(null);
+  const [currentStep, setCurrentStep] = useState(1);
```

Comment:
Right now, it is still somewhat fine. If this gets more complicated, I suggest you abstract even the numbers to some type. So you can keep track of what `2` step is. 

![CleanShot 2025-01-15 at 10 43 35](https://github.com/user-attachments/assets/6b0450c3-f0c8-4180-89a1-c8854e5c2777)

with

```
const STEPS = {
  NAME: 1,
  DESCRIPTION: 2,
};```